data-utilities/process-wichita-budget.js

```diff
@@ -19,6 +21,9 @@ fs.createReadStream(process.argv[2], { encoding: 'utf8' })
       return row.concat(['account_type', 'fiscal_year'])
     }
 
+    // make all description titles title-case
+    row[oca_title] = titleCase(row[oca_title])
```

Comment:
I think we need to title case more than just the `oca_title` here. All the fields listed in the `hierarchy` key in `_treemap/config.json` are displayed.